### ISSUE TITLE
Make a portable solution usable from shell

### ISSUE THREAD

**Issue by adlibdalom** [balthamos/geektool-3#10]
Hello!
First of all, thanks a lot for this tool, it seems really useful! I do, however, have a development request :-)
I'd love for a variant of this to have command line flags, so that I can just have the .app anywhere, and then call `NerdTool -use <path to log(s) I want to use>`

That way I can create a script like:
`cp -R "NerdTool.app" /path/to/where-I-want-the-app`
`/path/to/NerdToolRO.app/Contents/MacOS/NerdToolRO -use /path/to/log(s) &`
to launch a process, displaying the wanted information on a stand-alone computer without having to populate the user's /Application Support etc.
Thanks in advance!

**Comment by balthamos** [comment]
So what use case are you trying to go for? Multiple config files? Usage of NerdTool on machines other than your own?

**Comment by adlibdalom** [comment]
Yeah, I'm looking for a solution where I can project info on the desktop while I'm running an image restore process to install a fresh OS. I'm using DeployStudio and would like certain information items displayed during the restoration process.



    _____________________________

**Comment by balthamos** [comment]
Nerdtool hasn't been actively developed for quite some time. You should be able to mimic the desired functionality by having a script go and create the files Nerdtool is looking for and then delete them once the program is running.

I realize this is not a terribly elegant solution, but it should work.

In future development (whenever that may be), I'll be sure to take your request into consideration for a cleaner solution.
